packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -189,7 +189,7 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
       },
       getPlainContent() {
         if (editor.current) {
-          return exportContent(editor.current, 'PlainTextFast');
+          return exportContent(editor.current, 'PlainText');
```

Comment:
Could you addd a comment why `PlainTextFast` doesn't work here?
Can we get some issues with how fast the strings are processed in RichTexSendBox because of this change?

Reply:
Oh I need to revert this; I was checking whether this improved things or not.